src/clj/rems/auth/oidc.clj

```diff
@@ -164,10 +164,10 @@
                                 :body
                                 json/parse-string))
                 researcher-status (ga4gh/passport->researcher-status-by user-info)
-                id-data (merge id-data user-info)
-                user (find-or-create-user! id-data researcher-status)]
+                user-data (merge id-data user-info researcher-status)
+                user (find-or-create-user! user-data)]
             (when (:log-authentication-details env)
-              (log/info "logged in" id-data user-info user))
+              (log/info "logged in" user-data user))
```

Comment:
Should we keep the printing of the separate structures here, as it would show the situation before the merge, so you can see where each value came from? This is only for debugging IdP issues so it could be useful to know what happens in that level of detail.

Reply:
I agree with this.